sdcm/provision/scylla_yaml/scylla_yaml.py

```diff
@@ -65,6 +65,7 @@ def set_endpoint_snitch(cls, endpoint_snitch: str):
     rpc_interface_prefer_ipv6: bool = None  # False
     # [SeedProvider(class_name='org.apache.cassandra.locator.SimpleSeedProvider')]
     seed_provider: List[SeedProvider] = None
+    force_schema_commit_log: bool = None  # False
```

Comment:
interesting we missed that...

Reply:
see
https://github.com/scylladb/scylladb/issues/16427